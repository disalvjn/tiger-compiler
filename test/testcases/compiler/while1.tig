(while 0 do printInt(999) ; /* shouldn't print anything */
 let var i := 0
 in while i < 4 do
          let var j := 0
          in while j < 3 do
                (j := j+1;
                 if i > 2 then break else (printInt(i); printInt(j)));
             i := i + 1
          end
  end)
