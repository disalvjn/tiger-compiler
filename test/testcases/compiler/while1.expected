010203111213212223
